src/ets/scala/io/iohk/ethereum/ets/vm/testopcodes.scala

```diff
@@ -0,0 +1,79 @@
+package io.iohk.ethereum.ets.vm
+
+import akka.util.ByteString
+import io.iohk.ethereum.domain.Address
+import io.iohk.ethereum.vm._
+
+case object TestCREATE extends CreateOp {
```

Comment:
`TestCREATE` never creates new accounts and does not execute any initialisation code. It does however consume some amount of gas and can result in errors for certain argument values. It always replaces the real `CREATE` (probably because cpp-ethereum VM cannot operate in in-memory mode) and we only care about tracking them via `InternalTransaction` / `CallCreate`.